ui/src/app/shared/components/checkbox-filter/checkbox-filter.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import {Checkbox} from 'argo-ui';
+
+require('./checkbox-filter.scss');
+
+export class CheckboxFilter extends React.Component<
+    {
+        items: {name: string; count: number}[];
+        type: string;
+        selected: string[];
+        onChange: (selected: string[]) => void;
+    },
```

Comment:
Don't declare the props anonymously. Create an interface with a name. There are examples of this on the codebase.

Reply:
In fact, you seem to have done it correctly in other files in this PR

Reply:
👍